cs_util/tests/test_cfis.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+
+"""UNIT TESTS FOR CFIS SUBPACKAGE.
+
+This module contains unit tests for the cfis subpackage.
+
+"""
+
+import os
+
+import numpy as np
+from numpy import testing as npt
+from astropy import units
+
+from unittest import TestCase
+
+from cs_util import cfis
+
+
+class CfisTestCase(TestCase):
+    """Test case for the ``cfis`` module."""
+
+    def setUp(self):
+        """Set test parameter values."""
+
+        self._size_tile = 0.5 * units.deg
+
+        self._tile_number_ok = ['270.283', '188-308']
+        self._nix = ['270', '188']
+        self._niy = ['283', '308']
+        self._dec = [51.5, 64]
+        self._ra = [216.86237, 214.43017]
+        self._unit = units.deg
+
+        self._tile_number_nok = '12x.456'
+
+    def tearDown(self):
+        """Unset test parameter values."""
+        self._tile_number_ok = None
+        self._nix = None
+        self._niy = None
+        self._dec = None
+        self._ra = None
+        self._unit = None
+        self._tile_number_nok = None
+
+    def test_Cfis(self):
+        """Test ``cs_util.Cfis`` class.
+
+        """
```

Comment:
I think pydocstyle will complain about this. It likes one-line docstrings on one-line e.g. 

```python
"""Test ``cs_util.Cfis`` class."""
```

Reply:
Fixed